tests/test_post_gen_project.py

```diff
@@ -9,40 +10,105 @@
     get_repo_dir,
 )
 
+@pytest.mark.parametrize("repos",[
+    # test get_repo_dir when "scikit-package-manuscript" repo exists.
+    # C1: with only "scikit-package-manuscript" repo
+    ("scikit-package-manuscript",),
+    # C2: with "scikit-package-manuscript" and other repos
+    ("scikit-package-manuscript", "foo", "bar"),
+]
+)
+def test_get_repo_dir(user_filesystem, repos):
+    home_path, ck_path, cwd_path = user_filesystem
+    skm_repo_path = home_path / ".cookiecutters" / "scikit-package-manuscript"
+    os.environ["HOME"] = str(home_path)
+    for repo in repos:
+        repo_path = ck_path / repo
+        repo_path.mkdir()
+    actual = get_repo_dir()
```

Comment:
let's also rename this function and change its signature, something like: `get_cookiecutter_dir(cookiecutter_name="scikit-package-manuscript")`

We can then test it getting a different cookiecutter in general.